### PR TITLE
Upgrade kontainer version

### PR DESCRIPTION
**Pull request by orangedeng**

### CODE REVIEW

**Comment by gitlawr** [comment]
LGTM

File: driver.go [hunk 1]
```diff
@@ -592,3 +592,53 @@ func (d *CCEDriver) setNodeCount(ctx context.Context, clusterInfo *types.Cluster
...
+
+func (d *CCEDriver) ETCDSave(ctx context.Context, clusterInfo *types.ClusterInfo, opts *types.DriverOptions, snapshotName string) error {
+	return fmt.Errorf("ETCD backup operations are not implemented")
+}
+
+func (d *CCEDriver) ETCDRestore(ctx context.Context, clusterInfo *types.ClusterInfo, opts *types.DriverOptions, snapshotName string) error {
+	return fmt.Errorf("ETCD backup operations are not implemented")
+}
+
+func getClientSet(ctx context.Context, clusterInfo *types.ClusterInfo) (*kubernetes.Clientset, error) {
```
Comment (nathan-jenan-rancher) [inline driver.go:580]: This logic is duplicated in `PostCheck`, please refactor out the common logic (I'd probably have `PostCheck` call this func).

**Comment by orangedeng** [comment]
Updated

**Comment by nathan-jenan-rancher** [comment]
LGTM

**Comment by cjellick** [comment]
No isue is referenced. i dont know what to update

**Pull request merged**